### ISSUE TITLE
Modify CSV generation from Search in Samples results

### ISSUE THREAD

**Issue by mkoo** [AmphibiaWeb/amphibian-disease-tracker#220]
The ideal CSV file has species summary table values:
species, tally samples, tally positive samples, tally negative samples, percent positive detection

This would also be an ideal persistent dashboard table.

A little further explanation: the current Download Samples file (eg adp-global-search-result-data_1487026742516.csv) from this query is giving only a sample of the data (probably the first row for each species record) which is meaningless by itself. 

Basically the expectation would be a download of the same info we see in the table for SEearch in Summary results:
![screenshot-amphibiandisease org 2017-02-13 15-05-56](https://cloud.githubusercontent.com/assets/2523089/22907435/f97b676e-f1fd-11e6-9961-339f01c2d8c1.png)

**Comment by mkoo** [comment]
This may be a separate issue-- doing a a continent-wide Search in Samples or Search in Projects is giving a No Results error, despite me looking at RED DOTS on the map!

**Comment by tigerhawkvok** [comment]
Fixed at 23d7ee14b49e206af9045ee44cd4067b4dc81ab5
